### ISSUE TITLE
How can  I use it in iOS project？

### ISSUE THREAD

**Issue by syg1128** [sorz/asstosrt#5]
How can  I use it in iOS project？

**Comment by anonymous2ch** [comment]
Yes, MIT License generally gives you the right do almost anything, except blaming original developers for your own faults :)
